core/src/main/java/org/apache/spark/shuffle/sort/ShuffleInMemorySorter.java

```diff
@@ -148,7 +157,12 @@ public void loadNext() {
    * Return an iterator over record pointers in sorted order.
    */
   public ShuffleSorterIterator getSortedIterator() {
-    sorter.sort(array, 0, pos, SORT_COMPARATOR);
-    return new ShuffleSorterIterator(pos, array);
+    int offset = 0;
+    if (useRadixSort) {
+      offset = RadixSort.sort(array, pos, 5, 7, false, false);
```

Comment:
For my understanding. The 5-7 means that we are sorting longs based on their least significant 4-bytes? Perhaps add some documentation here.